### PR TITLE
Fix stars and forks.

### LINKED ISSUE #356: Fix Github API using features in page.
The stars and forks are not being shown in the website. Also the Issues are also not being loaded.There is some fault in the Github API calling.Once Fixing that problem will solve this issue.

### PR DESCRIPTION
**Pull request by Ace-Bansal**
Fixes #356

[link](https://ace-bansal.github.io/OpenCode-Collaborative/)

Changes proposed in this pull request:
- Earlier the number of stars and forks weren't being displayed correctly. Fixed that.

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@Ace-Bansal Fix looks good. But can you try the design that is currently there on website? If that's not possible, no problem, we can go with this one.

**Comment by Ace-Bansal** [comment]
@pradeepgangwar Sure I'll work on that.

**Comment by Anupam-dagar** [comment]
I still can't see the number of stars and fork, is it the expected behavior?Also looks like your fork got messed up and it's master branch too, consider re-forking again after this PR get merged.

**Comment by pradeepgangwar** [comment]
@Anupam-dagar It was working before, when i saw. @Ace-Bansal can u check what's the issue.

**Comment by Ace-Bansal** [comment]
@pradeepgangwar @Anupam-dagar Yeah I'll look into it :-)

**Comment by Ace-Bansal** [comment]
screenshot 44
@pradeepgangwar @Anupam-dagar The number of stars and forks are being displayed correctly.
Also I don't think that these buttons could be styled because they use an online css file for their formatting. So when I tried to copy the contents of that file and include it as a new file into our source code, the site broke.

**Comment by Anupam-dagar** [comment]
screenshot from 2018-01-23 20-17-29
this is visible to me

**Comment by Anupam-dagar** [comment]
@pradeepgangwar @aswinzz somebody's PR broke the center alignment of grid again :|

**Comment by Ace-Bansal** [comment]
@Anupam-dagar Try now. Actually I've taken these buttons from [https://buttons.github.io](https://buttons.github.io)... And they weren't working even on this site a few minutes ago. But they're working now. Otherwise I'll try to look at some other method.

File: index.html [hunk 1]
```diff
@@ -1093,7 +1079,6 @@ <h4 class="contact_head">Mubaris NK</h4>
 <script src="js/contestant.js"></script>
 <script src="js/projects.js"></script>
 <script src="js/issue-loader.js"></script>
-<script src="js/leaderBoard.js"></script>
```
Comment (pradeepgangwar) [inline index.html]: You actually deleted this new content present on site. Also there may be many such instances in code which are hard to figure out due to lot of changes.

Comment (Ace-Bansal) [reply index.html]: @pradeepgangwar I'll refork amd make a new PR... You may close this one.

**Comment by pradeepgangwar** [comment]
Closing on request.

**Pull request closed without merging**